Yugabyte support in YCSB.

psudheer21 opened the pull request:
This change adds 2 modules 'ybjdbc' and 'ybcassandra' to benchmark our DB using YCSB.

Running postgres benchmarks against our DB is as simple as:
bin/ycsb load ybjdbc -P ./db.properties -P workloads/workloada
bin/ycsb run ybjdbc  -P ./db.properties -P workloads/workloada

Running cassandra benchmarks against our DB is as simple as:
bin/ycsb load ybcassandra-cql -P db.properties -P workloads/workloada
bin/ycsb run ybcassandra-cql -P db.properties -P workloads/workloada

Each of the benchmarks use our drivers: 'jdbc-yugabytedb' for jdbc and 'cassandra-driver-core-yb' for cassandra.

psudheer21 committed:
Yugabyte support in YCSB.

Summary:
Added new targets 'ybjdbc' and 'ybcassandra-cql' that can be used to
benchmark yugabyte platforms.

All the options for the benchmark remain the exact same except for the
module name which changes from 'jdbc' to 'ybjdbc' in case of postgres
and from 'cassandra-cql' to 'ybcassandra-cql'.

So instead of running
bin/ycsb load jdbc -P ./db.properties -P workloads/workloada
against our DB we can run
bin/ycsb load ybjdbc -P ./db.properties -P workloads/workloada

Similarly for cql we would use:
bin/ycsb load ybcassandra-cql -P db.properties -P workloads/workloada

Reviewers: Karthik, Minhea

m-iancu commented on ybcassandra/pom.xml:
```diff
@@ -0,0 +1,147 @@
+    <dependency>
+      <groupId>com.yugabyte</groupId>
+      <artifactId>cassandra-driver-core</artifactId>
+      <version>3.2.0-yb-12</version>
```
Let's use the latest version `3.2.0-yb-19 ` (https://search.maven.org/artifact/com.yugabyte/cassandra-driver-core/3.2.0-yb-19/bundle)

psudheer21 replied:
Will do.

psudheer21 committed:
Renamed ybjdbc to yugabyteSQL and ybcassandra to yugabyteCQL

Summary:
This change has the renaming of the bindings.

It also has the README udpated for both the bindings.

Reviewers: Karthik, Mihnea

psudheer21 commented:
@rkarthik007 and @m-iancu, I have done the required changes as mentioned above.

I have also updated the README files for both:
https://github.com/yugabyte/YCSB/tree/NewLib/yugabyteCQL
https://github.com/yugabyte/YCSB/tree/NewLib/yugabyteSQL

Please take a look thanks.

m-iancu reviewed (approved):
LGTM!
